src/configs/strings/th.json

```diff
@@ -830,5 +830,9 @@
   "ORGANIZATION_NAME_TITLE": {
     "message": "ทีมปฏิบัติงาน",
     "description": "Organization list column header"
+  },
+  "WELCOME_BACK": {
+    "message": "",
+    "description": "prompt after a user has logged in"
```

Comment:
Let's remove this until we have an actual translation so it falls back to English. With this it will render an empty string instead of `Welcome Back` when run with TH translations.

Reply:
(I'll make a ticket)

Reply:
makes sense, lemmy remove it